rest/route/distro.go

```diff
@@ -50,13 +51,16 @@ func (h *distroIDGetSetupHandler) Parse(ctx context.Context, r *http.Request) er
 
 // Run returns the given distro's setup script.
 func (h *distroIDGetSetupHandler) Run(ctx context.Context) gimlet.Responder {
-	d, err := data.FindDistroById(h.distroID)
+	d, err := distro.FindOne(distro.ById(h.distroID))
```

Comment:
can use FindOneID

Reply:
Created and refactored all usages of this.